holoviews/plotting/renderer.py

```diff
@@ -301,15 +301,16 @@ def get_widget(self_or_cls, plot, widget_type, **kwargs):
         if not isinstance(plot, Plot):
             plot = self_or_cls.get_plot(plot)
         dynamic = plot.dynamic
+        # Whether dimensions define discrete space
+        discrete = all(d.values for d in plot.dimensions)
```

Comment:
This seems to be the basic condition that determines when scrubber can be used (with ``DynamicMap``).